pkg/standalone/list.go

```diff
@@ -180,7 +180,10 @@ func List() ([]ListOutput, error) {
 				listRow.Command = utils.TruncateString(run.appCmd, 20)
 			}
 
-			list = append(list, listRow)
+			// filter all dapr instances that do not start sidecar
+			if listRow.HTTPPort != 0 && listRow.GRPCPort != 0 {
```

Comment:
Is there a better way to detect only the dashboard process? I am asking this because we see instance where the sidecar does not start correctly and we end up with "zombie" CLI processes that we later on kill, so listing those "zombie" CLI processes is useful.